clients/drcachesim/simulator/cache_replacement_policy.h

```diff
@@ -71,7 +73,7 @@ class cache_replacement_policy_t {
     }
     /// Informs the replacement policy that an access has occurred.
     virtual void
-    access_update(int set_idx, int way) = 0;
+    access_update(int set_idx, int way, cache_access_outcome_t access_type) = 0;
```

Comment:
FYI @derekbruening on this API change.

Reply:
1) Is it in the changelist: yes.
2) We'll need to update our own code when pulling this in: which is also true for https://github.com/DynamoRIO/dynamorio/pull/7605. So let's coordinate which goes in at which time to separate them.